source/agora/consensus/validation/Block.d

```diff
@@ -96,6 +100,7 @@ public string isInvalidReason (const ref Block block, Height prev_height,
     scope EnrollmentFinder findEnrollment, size_t active_enrollments, size_t enrolled_validators,
     Point delegate (Height, ulong) nothrow @safe getValidatorAtIndex,
     Point delegate (const ref Point) nothrow @safe getCommitmentNonce,
+    ulong prev_timestamp, ulong curr_timestamp, uint block_timestamp_tolerance_secs,
```

Comment:
Instead of using a `uint` for `block_timestamp_tolerance_secs `, could we use a `core.time : Duration` ?